pyshtools/shclasses.py

```diff
@@ -1428,10 +1456,64 @@ def _expandGLQ(self, zeros, **kwargs):
             zeros, weights = _shtools.SHGLQ(self.lmax)
 
         data = _shtools.MakeGridGLQ(self.coeffs, zeros, norm=norm,
-                                    csphase=self.csphase, **kwargs)
+                                    csphase=self.csphase, lmax=lmax,
+                                    lmax_calc=lmax_calc)
         gridout = SHGrid.from_array(data, grid='GLQ', copy=False)
         return gridout
 
+    def _expand_coord(self, lat, lon, lmax_calc, degrees):
+        """Evaluate the function at the coordinates lat and lon."""
+        if self.normalization == '4pi':
+            norm = 1
+        elif self.normalization == 'schmidt':
+            norm = 2
+        elif self.normalization == 'ortho':
+            norm = 4
+        else:
+            raise ValueError(
+                "Normalization must be '4pi', 'ortho', or 'schmidt'. " +
+                "Input value was {:s}".format(repr(self.normalization)))
+
+        if degrees is True:
+            latin = lat
+            lonin = lon
+        else:
+            latin = lat * _np.pi / 180.
```

Comment:
`_np.deg2rad` (but I'm a bit confused by the meaning of `degrees` and this conditional.)